specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,128 @@
+# Background
+WebView2 has `ICoreWebView2AcceleratorKeyPressedEventArgs`  for the `AcceleratorKeyPressed` event. We have been asked to extend this API so that developers can indicate that they would like an existing browser accelerator key to be disabled as a browser accelerator.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending the `ICoreWebView2AcceleratorKeyPressedEventArgs` to allow developers to enable/disable the browser from handling accelerator keys with `AllowBrowserHandle` property. 
+
+# Examples
+## C++
+
+``` cpp
+    EventRegistrationToken m_acceleratorKeyPressedToken = {};
+
+    wil::com_ptr<ICoreWebView2Controller> m_controller;
+    wil::com_ptr<ICoreWebView2> m_webView;
+    wil::com_ptr<ICoreWebView2Settings3> m_settings3;
+
+  {
+        wil::com_ptr<ICoreWebView2Settings> settings;
+        CHECK_FAILURE(m_webView->get_Settings(&settings));
+        m_settings3 = settings.try_query<ICoreWebView2Settings3>();
+        if (m_setting3) 
+        {
+            // Disable all browser accelerator keys
+            CHECK_FAILURE(m_settings3->put_AreBrowserAcceleratorKeysEnabled(FALSE));
+            // Register a handler for the AcceleratorKeyPressed event.
+            CHECK_FAILURE(m_controller->add_AcceleratorKeyPressed(
+                Callback<ICoreWebView2AcceleratorKeyPressedEventHandler>(
+                    [this](
+                        ICoreWebView2Controller* sender,
+                        ICoreWebView2AcceleratorKeyPressedEventArgs* args) -> HRESULT
+                    {
+                        COREWEBVIEW2_KEY_EVENT_KIND kind;
+                        CHECK_FAILURE(args->get_KeyEventKind(&kind));
+                        // We only care about key down events.
+                        if (kind == COREWEBVIEW2_KEY_EVENT_KIND_KEY_DOWN ||
+                            kind == COREWEBVIEW2_KEY_EVENT_KIND_SYSTEM_KEY_DOWN)
+                        {
+                            UINT key;
+                            CHECK_FAILURE(args->get_VirtualKey(&key));
+
+                            wil::com_ptr<ICoreWebView2AcceleratorKeyPressedEventArgs2>
+                                args2;
+
+                            CHECK_FAILURE(args->QueryInterface(IID_PPV_ARGS(&args2)));

```

Comment:
This should probably gracefully handle the interface not being there, like we do for ICoreWebView2Settings3 above.